app/views/unattended/provisioning_templates/finish/preseed_default_finish.erb

```diff
@@ -31,6 +31,11 @@ oses:
 /bin/hostname  <%= @host.shortname %>.<%= @host.domain %>
 <% end -%>
 
+<% if @host.provision_method == 'image' && root_pass.present? -%>
+# Install the root password
+echo 'root:<%= root_pass -%>' | /usr/sbin/chpasswd -e
+<% end -%>
+
```

Comment:
This extra blank line is causing the snapshot failure, either we need to add another blank line in the snapshot or remove it from the template